src/com/mambu/apisdk/util/RequestExecutorImpl.java

```diff
@@ -114,20 +115,21 @@ public String executeRequest(String urlString, ParamsMap params, Method method,
 
 		}
 
+		HttpClient httpClient = new DefaultHttpClient();
```

Comment:
I'd prefer using the "try with resources" approach used in https://hc.apache.org/httpcomponents-client-4.5.x/quickstart.html or the fluent API which both take good care of releasing resources.